synapse/storage/data_stores/main/__init__.py

```diff
@@ -516,6 +516,18 @@ def get_users_paginate(self, order, start, limit):
         retval = {"users": users, "total": count}
         return retval
 
+    def get_user_count_txn(self, txn):
```

Comment:
this is removed in https://github.com/matrix-org/synapse/pull/5925/files#diff-13caa7216e6c08703de17985ecd65997L1625... perhaps you could leave it for now and we can try to land #5925?